libsql-server/src/connection/config.rs

```diff
@@ -29,6 +29,8 @@ pub struct DatabaseConfig {
     pub max_db_pages: u64,
     #[serde(default)]
     pub heartbeat_url: Option<Url>,
+    #[serde(default)]
+    pub bottomless_db_id: Option<String>,
```

Comment:
Have you considered adding something like `bottomless_url` instead?
It would give us the extra flexibility of decoupling the bottomless URL from the namespace name.
And we could reuse the concept, for example, in the call to fork databases from a different `libsql-server` cluster.

Reply:
bottomless_db_id is really a bottomless_url. Namespace name/id is appended to it before usage here (https://github.com/tursodatabase/libsql/pull/596/files#diff-8fb5af38655652ae31867fe58be95481a3dcfba6bfa8ddda5f43fd39a9a4fea9R731). I think we can reuse it for fork already, no?